src/edl/edl_initializer.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <stdint.h>
+
+// Extended Direct Media Layer
+namespace Tobot::EDL
```

Comment:
Don't really think an extra namespace is required here. If we use this enum only when initializing we could just put it inside `tobot_application.h`

Reply:
The enum should not be part of a class but we could declare it in the core namespace instead.

Reply:
The function EDLInitializerInitialize could of course be placed in the core, too.

Reply:
i didnt mean to move the enum into the class rather in the same file and namespace.